runtime/doc/vim2html.pl

```diff
@@ -1,16 +1,23 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 
 # converts vim documentation to simple html
 # Sirtaj Singh Kang (taj@kde.org)
 
 # Sun Feb 24 14:49:17 CET 2002
 
 use strict;
+use warnings;
 use vars qw/%url $date/;
 
 %url = ();
-$date = `date`;
-chop $date;
+# 30.11.23, Restorer:
+# This command does not work in OS Windows.
+# The "date" command in Windows is different from its counterpart in UNIX-like systems.
+# The closest analog is the "date /t" command, but how it would work in UNIX,
+# I don't know. I've corrected it as best I can. I don't know Perl.
+#$date = `date`;
+#chop $date;
+$date = localtime();
```

Comment:
Can we format the localtime() command more like a normal `date` output?

Reply:
`$date = localtime(); # outputs like this Fri Nov 3 00:56:59 2023`
Which, as I recall, is about the same as the "date" command with no arguments.
I made that now the date is output in the format DD.MM.YYYY.